frontend/lib/src/components/widgets/DataFrame/hooks/useCustomTheme.ts

```diff
@@ -75,9 +91,10 @@ function useCustomTheme(): CustomThemeReturn {
       textLight: theme.colors.fadedText40,
       textBubble: theme.colors.fadedText60,
       bgCell: theme.colors.bgColor,
-      bgCellMedium: theme.colors.bgColor, // uses same as bgCell to always have the same background color
-      cellHorizontalPadding: 8,
-      cellVerticalPadding: 3,
+      // uses same as bgCell to always have the same background color:
+      bgCellMedium: theme.colors.bgColor,
+      cellHorizontalPadding: convertRemToPx(theme.spacing.sm),
+      cellVerticalPadding: convertRemToPx("0.188rem"),
```

Comment:
We will probably change this to one of our predefined spacing variables in a follow-up PR.